test/factories/CommandFactoryTest.ts

```diff
@@ -17,12 +19,53 @@ describe("CommandFactory", () => {
 		aliases = new MockCommandWithAlias().getAliases();
 
 		factory = new CommandFactory();
-		factory["commands"] = {};
-		factory["commands"][commandName] = () => new MockCommand();
-		factory["commands"][commandWithAliasesName] = () => new MockCommandWithAlias();
+		factory.commands = {};
+		factory.commands[commandName] = () => new MockCommand();
+		factory.commands[commandWithAliasesName] = () => new MockCommandWithAlias();
 
```

Comment:
Oops. Only realized this now, I must have done it while using "Fix all auto-fixable problems". Let me know if you want me to undo it

Reply:
Nah, `factory.commands` makes more sense then `factory["commands"]`.